### PR TITLE
Deduplicate already existing ontology code and relevant test cases

### PR DESCRIPTION
**Pull request by hunterhector**
This PR refines the ontology generation module by:
1. Check already installed ontology codes (delivered with Forte), and make sure they don't get regenerated again.
2. Add test cases to cover these changes.

This PR is to resolve https://github.com/asyml/forte/pull/177

### CODE REVIEW

File: docs/ontology_generation.md [hunk 1]
```diff
@@ -1,13 +1,63 @@
 # Ontology Configuration #
 
-Welcome! Forte uses _Ontologies_ to allow interaction between different NLP 
-concepts. The ontologies are, simply put, a description of concepts and how they 
-relate to each other. 
+Welcome! Forte uses allows one to define relations between the annotations,
```
Comment (mgupta1410) [inline docs/ontology_generation.md]: typo - should be "Forte ~~uses~~ allows one"

File: docs/ontology_generation.md [hunk 2]
```diff
@@ -32,7 +82,7 @@ associated with an entry, like, `pos_tag` for the entry `Token`.
 in the module `forte.data.ontology.base.top`. All user-defined entries should extend one of 
 the top entries. 
  
-<!--We plan to deprecate base_ontology.py?-->
+<!--We plan to deprecate base_ontology_bak.py?-->
```
Comment (mgupta1410) [inline docs/ontology_generation.md]: Can be removed now that we are generating `base_ontology.py`

File: examples/chatbot/chatbot_example.py [hunk 3]
```diff
@@ -20,9 +20,8 @@
 from forte.data.readers import MultiPackTerminalReader
 from forte.common.resources import Resources
 from forte.pipeline import Pipeline
-from forte.processors import MicrosoftBingTranslator, BertBasedQueryCreator, \
-    SRLPredictor
-from forte.processors.search_processor import SearchProcessor
+from forte.processors import MicrosoftBingTranslator, SRLPredictor
+from forte.processors.ir import SearchProcessor, BertBasedQueryCreator
 from forte.data.selector import NameMatchSelector
 from forte.processors.nltk_processors import \
```
Comment (mgupta1410) [inline examples/chatbot/chatbot_example.py]: Can remove `\` at the end.

File: forte/data/multi_pack.py [hunk 4]
```diff
@@ -48,6 +50,9 @@ def __init__(self):
         super().__init__()
 
 
+# pylint: disable=too-many-public-methods
+
+# TODO: operations for multi pack is far less complete comparing to data pack.
```
Comment (mgupta1410) [inline forte/data/multi_pack.py]: Could you please open an issue for this specifying what operations are remaining?

File: ft/onto/wikipedia.py [hunk 5]
```diff
@@ -1,128 +1,210 @@
...
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from forte.data import DataPack
-from forte.data.ontology import Annotation, Generics
-from ft.onto.base_ontology import Document
-
-
-class WikiPage(Document):
+# ***automatically_generated***
```
Comment (mgupta1410) [inline ft/onto/wikipedia.py]: Wondering if we should have the copyright in the generated files.

**Comment by hunterhector** [comment]
Close due to various conflicts, will create a new PR

**Pull request closed without merging**